src/jobs/update_cards.ts

```
@@ -343,7 +343,7 @@ function convertType(card: ScryfallCard, preflipped: boolean, faceAttributeSourc
     type = 'Artifact Contraption';
   }
 
-  if (!type) {
+  if (!type && card.id !== 'custom-card') {
```

Suppress an error in the update cards